src/App.jsx

```diff
@@ -1,31 +1,33 @@
-import axios from 'axios'
-import { useEffect, useState } from 'react'
-import './App.css'
-import { Home } from './pages/Home/Home';
-import { Welcome } from './pages/Welcome/Welcome';
+import axios from "axios";
+import { useEffect, useState } from "react";
+import "./App.css";
+import { Home } from "./pages/Home/Home";
+import { Welcome } from "./pages/Welcome/Welcome";
 
 function App() {
-  const [photo, setPhoto] = useState('');
+  const [photo, setPhoto] = useState("");
   const user = localStorage.getItem("userName");
-  useEffect(() => { 
+  useEffect(() => {
     try {
       (async () => {
-        const {data} = await axios.get("https://api.unsplash.com/photos/random/?client_id=HbF7nKqBNGr5fl1Rhj2xYLNxhpWqFW1URj19bDYuiZM&query=aesthetic,landscape,adventure,travel&orientation=landscape");
-        setPhoto(data.urls.regular + "&w=1920&h=1080"); 
-      })()
+        const { data } = await axios.get(
+          "https://api.unsplash.com/photos/random/?client_id=HbF7nKqBNGr5fl1Rhj2xYLNxhpWqFW1URj19bDYuiZM&query=aesthetic,landscape,adventure,travel&orientation=landscape"
```

Comment:
it would be good if you get this url or just the key from the `.env` file instead of adding it directly in code.